theano/tensor/elemwise.py

```diff
@@ -1746,19 +1732,14 @@ class Any(CAReduce):
     Equivalent to `CAReduce(scalar.or\_, axis=axis)`.
 
     """
+    __props__ = ("axis", )
 
     def __init__(self, axis=None):
         CAReduce.__init__(self, scalar.or_, axis)
 
     def _output_dtype(self, idtype):
         return "int8"
 
-    def __str__(self):
```

Comment:
Removing this change the printing from Any{0,1} to Any{axis=[0,1]}.

The long explanation is that now we use metaclass. So if a subclass implement props, but not \__str__, we won't use the parent \__str__, but the default \__str__. I think it is better to not change the \__str__ for this class, so just add this back.